src/webapp/components/landing-page-edit-dialog/LandingPageEditDialog.tsx

```diff
@@ -278,6 +351,9 @@ export const LandingPageEditDialog: React.FC<LandingPageEditDialogProps> = props
     );
 };
 
+const FAVICON_ASPECT_RATIO = 1;
+const FAVICON_MAX_SIZE = 128;
```

Comment:
I would increase the max size of the favicon, as normally we even use 512x512 for favicons frequently in web environment ([MDN: Define your app icons](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/How_to/Define_app_icons))

Reply:
when i increase the size of the favicon, i can't view it in the browser and it defaults to the current page's icon

Reply:
I've checked in Chrome, Edge, and Firefox, and it works even with 1024x1024... we should ask a third person @adrianq  to check if user is able to see favicons bigger than 128